coderd/database/dbauthz/setup_test.go

```diff
@@ -198,11 +200,9 @@ func (s *MethodTestSuite) Subtest(testCaseF func(db database.Store, check *expec
 				s.Equal(len(testCase.outputs), len(outputs), "method %q returned unexpected number of outputs", methodName)
 				for i := range outputs {
 					a, b := testCase.outputs[i].Interface(), outputs[i].Interface()
-					if reflect.TypeOf(a).Kind() == reflect.Slice || reflect.TypeOf(a).Kind() == reflect.Array {
-						// Order does not matter
-						s.ElementsMatch(a, b, "method %q returned unexpected output %d", methodName, i)
-					} else {
-						s.Equal(a, b, "method %q returned unexpected output %d", methodName, i)
+					// Use cmp.Diff to get a nice diff of the two values.
+					if diff := cmp.Diff(a, b, cmpopts.EquateEmpty()); diff != "" {
+						s.Failf("compare outputs failed", "method %q returned unexpected output %d (-want +got):\n%s", methodName, i, diff)
 					}
```

Comment:
One thing we lose though is that the order should not matter for slices.

I was using `ElementsMatch` before. Does different ordering now break the comparison?

Reply:
That's a good point, there is [`cmpopts.SortSlices`](https://pkg.go.dev/github.com/google/go-cmp@v0.6.0/cmp/cmpopts#SortSlices) but it seems a bit more work to support than `ElementsMatch`. 🤔

TBH I'd prefer if all output from the DB was deterministic but that's a huge undertaking to fix it everywhere 😅

Reply:
Does [`6d2b94d` (#16161)](https://github.com/coder/coder/pull/16161/commits/6d2b94dcb531ec052b23175d179c9518c04923ee) seem acceptable?

Reply:
Works for me! 

Might be worth throwing in one of the test util packages.